CMakeLists.txt

```diff
@@ -74,6 +74,7 @@ elseif(COMPDEFS MATCHES "(.*)STM32F4xx(.*)")
     src/io/platform/f4xx/SPIf4xx.cpp
     src/dev/platform/f4xx/IWDGf4xx.cpp
     src/dev/platform/f4xx/RTCf4xx.cpp)
+
```

Comment:
You can leave this if you don't feel like fixing it, but I really prefer we don't make changes that aren't necessary
```suggestion
```